### PR TITLE
Gradio Local Editor cookbook

### PR DESCRIPTION
**Pull request by saqadri**
Gradio Local Editor cookbook

---
Stack created with [Sapling](https://sapling-scm.com). Best reviewed with [ReviewStack](https://reviewstack.dev/lastmile-ai/aiconfig/pull/826).
* __->__ #826
* #866
* #863

### CODE REVIEW

File: cookbooks/Gradio/README.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# Gradio Workbook Editor
+
+## Prompt IDE with LastMile AI
+
+This cookbook is meant to demonstrate the capabilities of the AI Workbook editor. It can run inference against locally hosted or remote models from many inference providers, including Hugging Face, OpenAI and others.
```
Comment (rholinshead) [inline cookbooks/Gradio/README.md:5]: Should "AI Workbook editor" be "local AIConfig Editor"?

File: cookbooks/Gradio/README.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+It supports text, image and audio model formats, allowing you to easily chain them together in a single notebook session!
+
+With `aiconfig`, it lets you save the state in a single json config file which you can version control or share with others.
```
Comment (rholinshead) [inline cookbooks/Gradio/README.md:9]: nit, "it lets you" --> "you can"

File: cookbooks/Gradio/README.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+**Note**: We already have the Gradio backend that corresponds to the Flask server in the [`gradio-workbook`](https://github.com/lastmile-ai/gradio-workbook) repo.
+
+We are working on using `sveltris` to package our React frontend to work with Gradio. Once that works, the same experience you see in this cookbook will be possible inside a Gradio custom component.
+
+## Getting Started
+
+**Instructions**:
+
+- Clone https://github.com/lastmile-ai/aiconfig
+- Go back to top-level directory: `cd <aiconfig>`
```
Comment (rholinshead) [inline cookbooks/Gradio/README.md:34]: nit, just "Go", no "back"

File: cookbooks/Gradio/README.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+```bash
+aiconfig edit --aiconfig-path=huggingface.aiconfig.json --parsers-module-path=hf_model_parsers.py
+```
+
+## TODO
+
+- Publish new version of aiconfig_extension_hugging_face package
+- Update huggingface.aiconfig.json with clean examples
+- Add video demo
```
Comment (rholinshead) [inline cookbooks/Gradio/README.md:56]: Can remove TODO

**Review by rholinshead — APPROVED** [review]

**Comment by rholinshead** [comment]
LGTM, just nits

**Comment by saqadri** [comment]
Will clean up readme afterwards. Thanks @rholinshead

**Pull request merged**